src/AriesSDK.js

```diff
@@ -65,6 +65,10 @@ sdk.scanIdentity = function(params) {
 	return invoke('scan.identity', params)
 }
 
+sdk.analyticsEvent = function(params) {
+	return invoke('analytics.event', parans)
```

Comment:
Typo!

Reply:
nice catch! though this is already merged, what do you guys usually do in these situations? commit straight to master or a new pull request fixing the typo?

Reply:
a new version hasn't been tagged  or released so just open a new PR and merge with master.